app/config/Module.scala

```diff
@@ -39,7 +41,8 @@ class Module extends AbstractModule {
     bind(classOf[IFSConnector]).to(classOf[IFSConnectorImpl])
     bind(classOf[IFSBusinessDetailsConnector]).to(classOf[IFSBusinessDetailsConnectorImpl])
     bind(classOf[MDTPConnector]).to(classOf[MDTPConnectorImpl])
-    bind(classOf[HipConnector]).to(classOf[HipConnectorImpl])
+    bind(classOf[BroughtForwardLossConnector]).to(classOf[BroughtForwardLossConnectorImpl])
```

Comment:
Removing the unnecesary traits will negate the need to create explicit module bindings here. Guice can automaticall find and inject instances of classes without the need for trait interfaces. Also because we're using Mockito instead of "Stub" classes, we don't need the traits.